packages/clerk-js/src/ui/components/SignUp/SignUp.tsx

```diff
@@ -21,8 +27,16 @@ function RedirectToSignUp() {
 }
 
 function SignUpRoutes(): JSX.Element {
+  const { __internal_setComponentNavigationContext } = useClerk();
+  const { navigate, basePath } = useRouter();
   const signUpContext = useSignUpContext();
 
+  usePreloadSessionTask(signUpContext.withSessionTasks);
+
+  React.useEffect(() => {
```

Comment:
Seems that now `bundlewatch` started to fail due to those new changes, let me perform a quick bundle analysis here just to make sure and then commit the bump on the limit